lib/webauthn/attestation.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+require "cbor"
+require "openssl"
+require "webauthn/attestation_statement"
+require "webauthn/authenticator_data"
+
+module WebAuthn
+  class Attestation
+    def self.deserialize(attestation_object)
+      from_map(CBOR.decode(attestation_object))
+    end
+
+    def self.from_map(map)
+      new(
+        authenticator_data: WebAuthn::AuthenticatorData.new(map["authData"]),
```

Comment:
This would change to `#deserialize` with https://github.com/cedarcode/webauthn-ruby/pull/302/files, which would make it more consistent, which is great.